scout/server/blueprints/variant/templates/variant/variant_details.html

```diff
@@ -152,8 +152,9 @@
 
 {% macro observations_panel(variant, observations, case) %}
   <div class="card panel-default">
-    <div class="panel-heading">
+    <div class="panel-heading d-flex justify-content-between">
       <a class="text-white" href="https://github.com/moonso/loqusdb" target="_blank">Local observations</a>
+      <span data-toggle="tooltip" title="Missing links to case variants might be caused by variants not loaded after a rerun">?</span>
```

Comment:
```suggestion
      <span data-toggle="tooltip" title="Nr of observations is the total seen in this loqusdb instance. Missing links on greyed out case names for shared local case variants might be caused by variants not loaded after a rerun or inexact SV matching. Only names of collaborator cases are shown. Please contact support to establish a contact for cases from other institutes.">?</span>
```

Reply:
Nooo no docs, let's keep it there. I think it's really helpful. Perhaps we could omit "Please contact support to establish a contact for cases from other institutes" because it's kind of obvious?

Reply:
Sure! That's also really the kind of knowledge that one would wish to have a separate channel for, as it is not really much to do with the software as such.